Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -432,6 +434,42 @@ private void AddCargoContents(CargoShuttleComponent component, StationCargoOrder
         }
     }
 
+    /// <summary>
+    /// In this method we are printing and attaching order manifests to the orders.
+    /// </summary>
+    private void spawnAndAttachOrderManifest(EntityUid item, CargoOrderData order, EntityCoordinates coordinates, CargoShuttleComponent component) {
```

Comment:
C# style guide please (pascal case method names).

Reply:
Will fix in the next commit. My bad, habit